### DISCUSSION TITLE
"Loading this dictionary failed"

### DISCUSSION THREAD

**Discussion by debrandell** [opensteno/plover#1594] [Q&A and Support]
I'm a new user of Plover and am using a Stentura 400 with it.  Yesterday, I began inputting strokes and translations into my user dictionary.  This morning, a restart was needed on my computer, and when I opened Plover, I got the message, regarding my user dictionary:  "Loading this dictionary failed: Expecting value line 1 column 1 (char 0)."  The main dictionary and command are fine.   Anyone know how to address or fix this? Thanks!

**Comment by debrandell** [comment]
I found there were no entries in the user.json file (0 KB), so I created a new user.json dictionary and replaced the old one with it. Not sure why I lost yesterday's work, but in the future, I'll quit the program before doing a reboot on the computer. Tested by quitting Plover, rebooting the computer, and reopening Plover, and the entries I did this morning in the user.json dictionary were there.

**Reply by aerickt** [reply]
Were you adding entries to the dictionary with Plover or editing the file itself (like in a text editor)?

**Comment by debrandell** [comment]
I had added several entries to the user dictionary, never to the file itself. I had not closed the dictionary when the computer rebooted, so I don't know if that had anything to do with the loss of data (user file showed 0 KB after rebooting). I named a new dictionary "user.json" and replaced the corrupted dictionary with it. I have been re-adding strokes to it today, closed it, rebooted the computer, and everything was there as it should be. Did this a couple times, and everything looked fine. Not sure what happened, but it's all good now. Thanks!

Sent using myEarthLink

On Thu Mar 02 14:02:43 EST 2023 Aerick wrote:

Were you adding entries to the dictionary with Plover or editing the file itself?

—
Reply to this email directly, view it on GitHub (https://github.com/openstenoproject/plover/discussions/1594#discussioncomment-5182381), or unsubscribe (https://github.com/notifications/unsubscribe-auth/A6GEE6MBZUYYWSD636UPPL3W2DVFHANCNFSM6AAAAAAVNTZSXY).
You are receiving this because you authored the thread.Message ID:

<openstenoproject/plover/repo-discussions/1594/comments/

5182381 (tel:5182381)

@github.com>

**Reply by aerickt** [reply]
I'm just trying to wonder how this issue came about. If you're editing the file itself, make sure to restart Plover after you're done with the edits.
> I had not closed the dictionary when the computer rebooted

I'm assuming you mean the dictionary editor window? In that case, that may have been the cause of your issue. Sometimes entries you add via that tool aren't saved until you close the window (though I'm not sure why).

**Reply by debrandell** [reply]
That is probably what happened. I don't believe I had closed the dictionary editor window before the computer was rebooted. Now I know! Thanks!

**Comment by debrandell** [comment]
I should ask, is there a problem with using a qwerty keyboard and a steno keyboard to edit and write in Word documents at the same time? I have them both plugged in and have been using them interchangeably. 😬

Sent using myEarthLink

On Thu Mar 02 14:02:43 EST 2023 Aerick wrote:

Were you adding entries to the dictionary with Plover or editing the file itself?

—
Reply to this email directly, view it on GitHub (https://github.com/openstenoproject/plover/discussions/1594#discussioncomment-5182381), or unsubscribe (https://github.com/notifications/unsubscribe-auth/A6GEE6MBZUYYWSD636UPPL3W2DVFHANCNFSM6AAAAAAVNTZSXY).
You are receiving this because you authored the thread.Message ID:

<openstenoproject/plover/repo-discussions/1594/comments/

5182381 (tel:5182381)

@github.com>

**Reply by aerickt** [reply]
Nope, there shouldn't be anything wrong with that.

**Reply by debrandell** [reply]
Thanks again!!!
